models/brandsetting/fields.yaml

```diff
@@ -4,10 +4,23 @@ tabs:
             label: winter.tailwindui::lang.branding.auth_layout.label
             tab: backend::lang.branding.navigation
             type: balloon-selector
+            span: left
             default: split
             options:
                 simple: winter.tailwindui::lang.branding.auth_layout.simple
                 split: winter.tailwindui::lang.branding.auth_layout.split
+        backgroundImage:
```

Comment:
This field will need to be hidden on the user's backend preferences field, just like the auth_layout field is. See Plugin.php ->extendBrandSettingsForm()

Reply:
done